src/types/User.ts

```diff
@@ -0,0 +1,6 @@
+export interface User {
+  id: number,
```

Comment:
Consider adding more properties to the User interface, such as `age`, `address`, `phone`, etc. 🏠